### PR TITLE
Ballen

### PR DESCRIPTION
**Pull request by gigline**
My pull request

### CODE REVIEW

File: ballen-bio.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+My Big Biographical Commitments
+
+
+Another line of text
+- Works for Younique
+- Lives in Tooele
+- USS Independence Sailor
+
+- Eats foods
```
Comment (obscurecloud) [inline ballen-bio.md:9]: Can you provide some proof of this claim, please?

File: ballen-bio.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+My Big Biographical Commitments
+
+
+Another line of text
+- Works for Younique
+- Lives in Tooele
```
Comment (kirtjMIL) [inline ballen-bio.md:6]: You call that living? :metal:

File: ballen-bio.md (hunk 1 above)
Comment (gigline) [inline ballen-bio.md:9]: The proof is in the pudding - so it's self evident

**Comment by gigline** [comment]
The lashes were for my wife, I swear!

**Pull request merged**